kie-soup-project-datamodel/kie-soup-project-datamodel-api/src/main/java/org/kie/soup/project/datamodel/oracle/ModuleDataModelOracle.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
```

Comment:
There is probably thousand of these. Do we keep this or update to 2018?

Reply:
I'm not sure if we have an established rule or best practice for this. If not, I would think we should leave the original date. This file is not substantially different than under its previous name, and we do not typically update the copyright year with every change to a file.